lib/net/modules/dns.toit

```diff
@@ -249,42 +319,49 @@ class DnsClient:
   static is-letter_ c/int -> bool:
     return 'a' <= c <= 'z' or 'A' <= c <= 'Z'
 
-  find-in-cache_ name --accept-ipv4/bool --accept-ipv6/bool -> net.IpAddress?:
-    if accept-ipv4:
-      if cache_.contains name:
-        entry := cache_[name]
-        if entry.valid: return entry.address
-        cache_.remove name
-    if accept-ipv6:
-      if cache-ipv6_.contains name:
-        entry := cache-ipv6_[name]
-        if entry.valid: return entry.address
-        cache-ipv6_.remove name
+  find-in-cache_ name type/int -> List?:
```

Comment:
static?

Reply:
Uses the cache, which is a field.  Same applies to trim-cache_.